packages/x-scheduler-headless/src/calendar-grid/day-event/CalendarGridDayEvent.tsx

```diff
@@ -78,12 +78,16 @@ export const CalendarGridDayEvent = React.forwardRef(function CalendarGridDayEve
   });
 
   const firstEventOfSeries = schedulerEventSelectors.processedEvent(store.state, eventId)!;
+
   const originalOccurrence: SchedulerEventOccurrence = {
     ...firstEventOfSeries,
     id: eventId,
     key: occurrenceKey,
-    start,
-    end,
+    displayTimezone: {
```

Comment:
Until we have a clean occurrence object that does not extend the event, I would update both `displayTimezone` and `dataTimezone` just to be safe.

A new `generateOccurrenceFromEvent` util method would probably be useful to handle that.